lib/table.c

```diff
@@ -324,6 +324,12 @@ struct route_node *route_node_get(struct route_table *table,
 	table->count++;
 	route_lock_node(new);
 
+	if (p->family == AF_FLOWSPEC){
```

Comment:
Yeah no.  This is just a crash waiting to happen.  This is not how to fix this problem

Reply:
It appears that the CI tests passed successfully.

To clarify, the reason for suggesting to check for the `AF_FLOWSPEC` condition when freeing the temporary memory allocated by `prefix_copy` is because the same condition is used for memory allocation. Furthermore, it's worth noting that the value stored in this temporary memory is subsequently copied to `node->p` within the `route_node_set` function, which means the temporary memory becomes unnecessary at that point.

Did I get that correctly? Thanks for your input!

Reply:
@keelan10, the good solution has to deal with a clean architecture. and what you do here it to mess flowspec with tables, which has nothing to do together.
I tried to do something without success . please see https://github.com/FRRouting/frr/pull/7485#issuecomment-734208339

Reply:
@pguibert6WIND You make a valid point. Thanks for the feedback